CHANGELOG.md

```diff
@@ -2,10 +2,15 @@
 
 ## v1.6.0 nf-core/mhcquant "Beijing Duck" - 2020/09/11
 
-### `Added`
+### `Fixed`
```

Comment:
You use the word "added", but list it under "fixed"? :)

Reply:
yeah that might be confusing - just Chris mentioned those captions should be more consistent in the master PR

Reply:
@christopher-mohr should I rather change it back to `Added` instead and leave `Fixed` empty?

Reply:
@Leon-Bichmann up to you, but what I meant is that you could have three categories `Added`, `Changed` and `Fixed` but have them for every release even though it might be empty. Btw why did you create a new PR? :)

Reply:
ok thought I have to create a PR to dev first

Reply:
Ah now I see, yes I guess that's better..